files/zh-cn/web/progressive_web_apps/guides/best_practices/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 渐进式 Web 应用的最佳实践
+slug: Web/Progressive_web_apps/Guides/Best_practices
+---
+
+{{PWASidebar}}
+
+[渐进式 Web 应用](/zh-CN/docs/Web/Progressive_web_apps)（PWA）可以安装在设备上，并能像传统网站一样在 Web 浏览器中使用，因此 PWA 需要能够适应不同的环境和用户期望。
+
+本文提供了一系列最佳实践，以帮助你确保你的 PWA 尽可能完善。
+
+## 适应所有浏览器
+
+你的 PWA 基于 Web 技术。这意味着除了可以安装在设备上之外，PWA 也可以在 Web 浏览器中运行。为了确保兼容性，非常重要的是在各种浏览器和操作系统上[测试你的应用](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing)。
+
+考虑到用户可能使用的各种浏览器，满足广泛的潜在用户需求。使用[功能检测](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection)可以让你为最广泛的受众提供可用的体验。
+
+功能检测还有助于实现{{Glossary("渐进增强")}}，这是一种设计理念，可以让尽可能多的用户获得出色的体验。
```

Comment:
The macro call here is not correct, please see the [implementation](https://github.com/mdn/yari/blob/bd9922925f9ed1c97c2ed6b0abef84b86c539119/kumascript/macros/Glossary.ejs#L11-L13).

The reason why the link in preview page is not "red" is that, the [old slug has been redirected to the new one](https://github.com/mdn/translated-content/blob/56981c5698d0c826274336efe25b9f45f6e3b905/files/zh-cn/_redirects.txt#L552). But please always use the newest link instead of the old one.

Documentation: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros

```suggestion
功能检测还有助于实现{{Glossary("Progressive Enhancement", "渐进增强")}}，这是一种设计理念，可以让尽可能多的用户获得出色的体验。
```